src/hotspot/os/linux/gc/z/zMountPoint_linux.cpp

```diff
@@ -88,14 +88,14 @@ void ZMountPoint::get_mountpoints(const char* filesystem, ZArray<char*>* mountpo
     }
   }
 
-  os::free(line);
+  ALLOW_C_FUNCTION(::free, ::free(line);) // *not* os::free
```

Comment:
Please extend comment: "readline will return malloc'd memory. Needs raw ::free, not os::free."